### ISSUE TITLE
Checkbox.Card in indeterminate state gives bad a11y attributes

### ISSUE THREAD

**Issue by SlamaFR** [mantinedev/mantine#9095]
### Dependencies check up

- [x] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

9.5.0

### What package has an issue?

@mantine/core

### What framework do you use?

React Router

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

`Checkbox.Card` does not appear to support the indeterminate state correctly from an accessibility perspective.

I’m trying to use `Checkbox.Card` in an indeterminate state, where the underlying checkbox should expose aria-checked="mixed". However, I cannot get the component to render the correct ARIA state. I tried setting/overriding `aria-checked`, but the value does not appear to be reflected correctly on the actual element with role `checkbox`.

As a result, when writing my tests, I can't assert that my checkbox is partially checked (using the `expect(...).toBePartiallyChecked()` from Testing Library)

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/devbox/mantine-react-template-forked-9w7zrp

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
Fixed in [9.5.1](https://github.com/mantinedev/mantine/releases/tag/9.5.1)

**Comment by SlamaFR** [comment]
Oh wow thank your very much for your prompt response! Everything is working great now 😄
